pytest.ini

```
@@ -1,2 +1,2 @@
 [pytest]
-addopts = --color=yes
+addopts = --color=yes --durations=0
```

```suggestion
addopts = --color=yes --durations=3
```

Should this be a non-zero number e.g. 3?

---

`--durations=0` means "dispaly all test durations". How about 30 which means "the slowest 30 tests"?

---

Ah got it, I think in practice we probably want this to be a pretty small number to avoid bloating test output when running locally - maybe 5? Since the intended use of this option is mainly to allow us to debug if we notice individual builds spontaneously lagging

---

Makes sense. I just fixed it to 5.